dnf.spec

```diff
@@ -32,6 +32,8 @@
 # Use the same directory of the main package for subpackage licence and docs
 %global _docdir_fmt %{name}
 
+%global common_desc As a Yum CLI compatibility layer, supplies /usr/bin/yum redirecting to DNF.
```

Comment:
could you rename the variable to something more descriptive?
e.g. yum_package_description